rllib/agents/ppo/ddppo.py

```diff
@@ -74,8 +74,6 @@
         "truncate_episodes": True,
         # This is auto set based on sample batch size.
         "train_batch_size": -1,
-        # Trajectory View API not supported for DD-PPO yet.
-        "_use_trajectory_view_api": False,
```

Comment:
We also need to set it to True in trainer.py right?

Reply:
Let's do this in a different PR. Too many contrib/non-functional-API algos would still break with the traj. view API on.
We can flip this flag then in that PR (turn on by default in trainer, then disable again in some algos). ..